osu.Android/AndroidMouseSettings.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Android.OS;
+using osu.Framework.Allocation;
+using osu.Framework.Android.Input;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Localisation;
+using osu.Game.Configuration;
+using osu.Game.Localisation;
+using osu.Game.Overlays.Settings;
+using osu.Game.Overlays.Settings.Sections.Input;
+
+namespace osu.Android
+{
+    public class AndroidMouseSettings : SettingsSubsection
+    {
+        private readonly AndroidMouseHandler mouseHandler;
+
+        protected override LocalisableString Header => MouseSettingsStrings.Mouse;
+
+        private Bindable<double> handlerSensitivity;
+
+        private Bindable<double> localSensitivity;
+
+        private Bindable<bool> relativeMode;
+
+        public AndroidMouseSettings(AndroidMouseHandler mouseHandler)
+        {
+            this.mouseHandler = mouseHandler;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager osuConfig)
+        {
+            // use local bindable to avoid changing enabled state of game host's bindable.
+            handlerSensitivity = mouseHandler.Sensitivity.GetBoundCopy();
+            localSensitivity = handlerSensitivity.GetUnboundCopy();
+
+            relativeMode = mouseHandler.UseRelativeMode.GetBoundCopy();
+
+            // High precision/pointer capture is only available on Android 8.0 and up
+            if (Build.VERSION.SdkInt >= BuildVersionCodes.O)
```

Comment:
Not sure if this should be hidden or disabled on unsupported platforms.
Went with hiding, as an unsupported device will never get updated to a newer version.

Reply:
I think hiding makes sense here yeah.